projects/handlefi/index.js

```diff
@@ -9,7 +9,9 @@ const abi = require("./abi.json");
 const transformArbitrumAddress = addr => `arbitrum:${addr}`
 const treasuryContract = "0x5710B75A0aA37f4Da939A61bb53c519296627994"
 const WETH_arbitrum = "0x82af49447d8a07e3bd95bd0d56f35241523fbab1"
-const treasuryTokens = [WETH_arbitrum]
+const FOREX_arbitrum = "0xDb298285FE4C5410B05390cA80e8Fbe9DE1F259B"
```

Comment:
Sorry, we usually do not count native tokens for tvl since it can be used to inflate the value

Reply:
so, if governance tokens are used as collateral for issuance of fxToken loans - this collateral is not counted? 

this appears inconsistent with other approaches taken; eg: for example SNX, where its staked SNX tokens staked are counted as TVL?

Reply:
Yeah, it is kinda fuzzy, thats why I wanted to check on your app how much of the TVL comes from forex, was surprised to that only ETH is supported. That is one reason that I was against it.

Counter example for SNX is mochi token was not count by us, the same way, I am having a discussion with smartcredit team because 95% of their tvl is native token

Reply:
appreciate the challenge, and the manipulation vectors - protocol  roadmap is multicurrency and only small % of TVL is related to FOREX, but would still like to be accurate? 

repultable project backers, probably mitigates these risks to a large extent?  https://docs.handle.fi/project-supporters

Reply:
@seahound  I have kinda tried your product before, so trusting you you and hoping my faith is not misplaced